etc/osm-smartrak.osm2pgsql.mml

```diff
@@ -465,6 +465,26 @@
       "srs": "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0.0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over", 
       "srs-name": "900913", 
       "status": "on"
+    },  
+      {
+      "Datasource": {
+        "dbname": "osm", 
+        "extent": "-20037508.34 -20037508.34 20037508.34 20037508.34", 
```

Comment:
**thought:** unrelated to this particular ticket, but I wonder if setting a more constrained `extent` could improve the performance of at least some parts of the map build. There's [documentation on the meaning](https://cartocss.readthedocs.io/en/latest/mml.html#extent-ogr-postgis-pgraster) of this field. On the other hand since we only try to render detailed tiles for a constrained region it might not make any difference.

Reply:
Ooh that would be interesting to experiment with! Even though the tiles are already constrained, it seems that updating the `extent` values would mean we'd store less data in the DB - maybe it would be quicker to load & query! 

Maybe experimenting with these values would make it more worthwhile to separate the MMLs for dotcom & skate.

Reply:
Hmm I thought it was `osm2pgsql` that stores the data in the database and these queries just get the data out? Still might be worth looking into, there's always a chance that narrowing the extent could make some queries faster.

Reply:
Okay that makes a lot of sense and brings up a separate question for me - is `osm2pgsql` a misnomer for this file? My assumption is yes but that it is what the underlying project we reference was using